pkgs/development/python-modules/typed-argparse/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  typing-extensions,
+}:
+buildPythonPackage rec {
+  name = "typed-argparse";
```

Comment:
```suggestion
  pname = "typed-argparse";
```

Reply:
I had completely misremembered which of these was preferred, thank you!